app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -271,15 +271,18 @@ def pod_definition
         }
       },
       :spec       => {
-        :restartPolicy => 'Never',
-        :containers    => [
+        :serviceAccount => 'management-admin',
+        :restartPolicy  => 'Never',
+        :containers     => [
           {
             :name            => "image-inspector",
             :image           => inspector_image,
             :command         => [
               "/usr/bin/image-inspector",
               "--image=#{options[:image_full_name]}",
-              "--serve=0.0.0.0:#{options[:pod_port]}"
+              "--serve=0.0.0.0:#{options[:pod_port]}",
+              "--username=serviceaccount",
+              "--password-file=/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
@enoodle I like constants for these in general (the password file path, the username).